src/Components/Nav/index.jsx

```diff
@@ -0,0 +1,19 @@
+import { useState } from "react"
+import NavLinks from "../NavLinks"
+
+function Nav() {
+    const [modalOpen, setModalOpen] = useState(false)
+
+    function handleClick() {
+        setModalOpen(!modalOpen)
+    }
+
+    return (
+        <div>
+            <button className='border px-3 rounded' onClick={handleClick}>MENU</button>
+            {modalOpen && <NavLinks />} 
```

Comment:
Now that NavLinks requires modelOpen and handleClick, you should pass them in as props